HOW_WE_USE_GITHUB.md

```diff
@@ -265,6 +267,81 @@ Community support can be found elsewhere, though, and we encourage you to explor
 
 In order to not have to manually type or copy/paste the above repeatedly, note that it's possible to add text for the most commonly-used responses via [GitHub's "Add Saved Reply" option][docs-saved-reply].
 
+## Development Processes
+
+The following are practices the conda organization encourages for feature
+development. While we recommend projects under the conda organization adopt
+these practices, they are not strictly required.
+
+### How should we approach feature development?
+
+For new features, first open an issue if one doesn’t exist. Once the feature
+has been accepted by the issue's status transitioning from the Sorting state to
+Refinement, create a specification to gather early feedback. This can include
+mockups, API/command references, a written plan in the issue, and sample CLI
+arguments (without functionality).
```

Comment:
```suggestion
For new features, first open an issue if one doesn’t exist. Once the feature request
has been accepted (indicated by the issue's status transitioning from "Sorting to
"Refinement"), create a specification to gather early feedback. This can include
mockups, API/command references, a written plan in the issue, and sample CLI
arguments (without functionality).
```